buildspec.yaml

```diff
@@ -6,6 +6,7 @@ phases:
       # Temporarily - install yarn if it's not there already
       - yarn --version || npm install --global yarn
       - yarn install --frozen-lockfile
+      - yum -y install gcc
```

Comment:
Apparently `testify` requires `gcc` to be available. I've added it to the `Dockerfile` as well so we'll have it baked into the image after the next release, at which point i'll remove this. 

One thing that bothers me is that testify officially only [supports](https://pkg.go.dev/github.com/stretchr/testify#readme-supported-go-versions) go `1.13` and `1.14`, and I suspect that with those versions, `gcc` is not required. I haven't dived deeper here though, and once `gcc` is installed, everything seems to be working properly. 

Happy to search alternatives if we have strong objections.

Reply:
Maybe add a comment to explain that... So we know when we're safe to pull the install out?

Reply:
Added